README.md

```diff
@@ -1 +1,195 @@
-# React_Sneakers_Store
\ No newline at end of file
+<div align="center">
+<h1 align="center">Games Gallery</h1>
+</div>
+
+
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+    - [🎥Walkthrough](#walk-though)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run tests](#run-tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Games Gallery] `React` `Redux` `API` `ES6`<a name="about-project"></a>
+
+
+**Games Gallery**  is a Single Page Application (SPA) designed for users to explore games by category and view details of their chosen games. We provide an exciting and up-to-date experience for gaming enthusiasts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li>Front-end: React, React Redux, React Router,JavaScript.</li>
+    <li>API Integration: Free-to-Play Games Database for real-time data.</li>
+    <li>Version Control: GitHub with Gitflow.</li>
+    <li>Testing: React Testing Library, Jest.</li>
+    <li>Build Tool: Create React App.</li>
+  </ul>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Home**
+  
+The Home section serves as the main landing page, offering users the ability to explore and filter games by category. It provides an overview of available game categories and allows users to click on a specific category to view details about games within that category. This section is designed for users to easily discover and select games based on their interests.
+
+- **Details**
+ 
+The Details section provides in-depth information about a selected game from the chosen category. Users can access details such as the game's title, description, platform availability, developer, release date, and more. It serves as a dedicated space for users to get comprehensive insights into the games they are interested in and make informed decisions about which games to explore further.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## :movie_camera: WALKTHROUGH <a name="walk-though"></a>
+
+- [Walkthrough video](https://www.loom.com/share/76e21ba7923c4b2ba57292405a3729f4?sid=4a88d68d-e2a4-47e0-b8b7-da91e6406356)
+
```

Comment:
- You have done great working on your project and also implementing most of the changes required by the previous review. :+1: But there are still some requirements you need to meet before moving to the next milestone. :100: 

- [ ] You did great adding the video as required and suggested by the previous reviewer, but looking at the requirement, it says `you should also highlight some interesting piece of code or something you built that you are very proud of.`. Going through your video presentation, you didn't do this. I'll suggest you make a video to show a piece of code you wrote that was challenging at first, or that works great or that liked how it turned out and add it to your video timeline. :smile: :small_airplane: 

- - Screenshot reference:
 
![Selection_017](https://github.com/JavierAybar/React_Games_Gallery/assets/107632780/fca41730-fb67-493c-a31d-f8e857fb28db)

Reply:
🙃Ok I will do it. Thank you for taking the time to review my project.